packages/suite/src/components/wallet/Fees/FeeDetails.tsx

```
@@ -1,120 +1,268 @@
+                                    amount={fee?.networkAmount || ''}
+                                    symbol={symbol}
+                                />
+                            }
+                            bottomRightChild={
+                                <>
+                                    {fee?.feePerUnit} {getFeeUnits(networkType)}
+                                    {hasInfo ? ` (${transactionInfo.bytes} B)` : ''}
+                                </>
+                            }
+                        />
+                    ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)